certora/specs/AccessControlDefaultAdminRules.spec

```diff
@@ -309,17 +317,19 @@ rule pendingDefaultAdminDelayEnforced(env e1, env e2, method f, calldataarg args
 */
 rule acceptDefaultAdminTransfer(env e) {
   require nonpayable(e);
-  requireInvariant defaultAdminConsistency(defaultAdmin());
-  requireInvariant singleDefaultAdmin(e.msg.sender, defaultAdmin());
```

Comment:
Have you run this rule? I think it's required because otherwise, the `e.msg.sender` may have the `DEFAULT_ADMIN_ROLE` without being the `defaultAdmin()`, otherwise the `acceptDefaultAdminTransfer` call will succeed without being called by the `pendingAdminBefore`

Reply:
I see. I assume then the prover is not considering the initial case of multiple default admins, which it does when specifying an `invariant`. Do you think that's the reason? I just don't want to assume that if it passes it's okay, I don't completely see why it passes.

Reply:
The `acceptDefaultAdminTransfer` is restricted to the `pendingDefaultAdmin`, not the `defaultAdmin`. It doesn't include any role check.

Then the `_acceptDefaultAdminTransfer` does :
- the schedule check
- `_revokeRole` 
  - if there is an inconsistency (default admin doesn't have the role) then this is a no-op that passes. Only thing we know for sure is the (old) defaultAdmin() doesn't have the role after
- `_grantRole`
  - if there is an inconsistency (pending default admin has the role) then this is a no-op that passes. Only thing we know for sure is the new defaultAdmin() has the role after
  
I don't see which part of the function call or the rule needs the defaultAdmin being consistent.

Reply:
My bad, I mistaken the check of the `pendingDefaultAdmin` by the `defaultAdmin` 👍🏻